account_reconcile_advance/model/account_reconcile_advance.py

```diff
@@ -314,31 +389,34 @@ def payment_reconcile(self, cr, uid, ids, context=None):
         adv_2_rec = []
         if used_aml_ids:
 
-            #            when reconciling among used_aml_ids, those ids should be grouped by
-            #            account_id, take care of the last used_aml_ids, because is with this
-            #            that the aml_sum (debit) should be created.
-            #            aml_grn_sum => credit
-            #            aml_sum => debit
+            #   when reconciling among used_aml_ids, those ids should be
+            #   grouped by account_id, take care of the last used_aml_ids,
+            #   because is with that the aml_sum (debit) should be created.
+            #   aml_grn_sum => credit
+            #   aml_sum => debit
             if aml_sum:
```

Comment:
can you check with original author whyc is commented this code?, if necesary delete it.